Products/ZenModel/skins/zenmodel/login_form.pt

```diff
@@ -1,140 +1,171 @@
+<!DOCTYPE html>
 <html>
     <head>
+        <meta charset="utf-8">
         <link rel="shortcut icon"
               type="image/x-icon"
               href="/zport/dmd/favicon.ico"/>
         <title>Login</title>
         <style>
+            * {
+                zoom: 1; /* IE7 compatibility */
+            }
```

Comment:
IE11 is the minimum version of IE that Zenoss supports.

Reply:
I can remove a code dedicated to IE11-, or even switch to flexbox for more flexibility